rust/tools/authorization-logic/src/parsing/AuthLogic.g4

```diff
@@ -25,20 +25,26 @@ grammar AuthLogic;
 //-----------------------------------------------------------------------------
 // Parser
 //-----------------------------------------------------------------------------
+id 
+    : VARIABLE #variable
+    | '"' CONSTANT_BODY '"' #constant
+    | '"""' MULTILINE_CONSTANT_BODY '"""'  #multilineConstant
```

Comment:
Not reviewing this as a code review, but out of interest:

When is it useful to have muilti-line identifier names? I can only imagine this causing confusion.

Alternatively, allowing multi-line string literals seems perfectly reasonable (though I see string literal is missing from `pred_arg` and possibly the file)

Reply:
Maybe "id" isn't the right term. This piece of syntax is really: principal names and predicate arguments. The different forms are either variables (which are similar to predicate arguments in souffle that are not wrapped in quotes) or the two different kinds of constants (which are similar to souffle arguments that *are* wrapped in quotes). I think when you're saying string literal you mean the same thing as what I am calling "constants" here.

You are right, it isn't useful to have miltiline identifiers. This syntax only permits multiple lines in the things that are constants (string literals).